packages/zapp/console/env.js

```diff
@@ -37,7 +37,7 @@ const STATUS_URL = process.env.STATUS_URL;
 
 // Configure Google Analytics
 const ENABLE_GA = process.env.ENABLE_GA || false;
-const GA_TRACKING_ID = process.env.GA_TRACKING_ID || 'G-0QW4DJWJ20';
```

Comment:
Maybe we should just disable GA by default, instead of configuring tracking id to be invaild.

Reply:
Looks like it is already default to `false`, so maybe this is just a documentation problem after all?

Reply:
It would still make sense to remove the Key even for cleaness or even security concerns if it's a valid GA key.